shaders/CompoundCloudPlane.shader

```diff
@@ -13,6 +13,7 @@ uniform vec4 colour4 : hint_color = vec4(0, 0, 0, 0);
 uniform vec2 UVOffset = vec2(0, 0);
 
 uniform float BrightnessMultiplier = 1.0f;
+uniform float CloudAlpha = 0.7f
```

Comment:
This looks like this is the line with a missing semicolon.

Reply:
sorry for this, tested it and it should work now.